R/bfast01classify.R

```diff
@@ -52,6 +52,7 @@
 #' @export bfast01classify
 
 bfast01classify <- function(object, alpha=0.05, pct_stable=NULL, typology=c("standard", "drylands")) { 
```

Comment:
```suggestion
bfast01classify <- function(object, alpha = 0.05, pct_stable = NULL, typology = c("standard", "drylands")) { 
```

since I'm here anyway...